src/utilities/device_utils.c

```diff
@@ -1618,87 +1713,6 @@ hypre_ResetDeviceRandGenerator( hypre_ulonglongint seed,
 
 #endif /* #if defined(HYPRE_USING_CURAND) || defined(HYPRE_USING_ROCRAND) || defined(HYPRE_USING_ONEMKLRAND) */
 
-/*--------------------------------------------------------------------
- * hypreGPUKernel_filln
- *--------------------------------------------------------------------*/
-
-template<typename T>
-__global__ void
-hypreGPUKernel_filln(hypre_DeviceItem &item, T *x, size_t n, T v)
```

Comment:
@victorapm All these fill functions should not have been moved out of this section, as they are needed for the sycl backend. I guess this was probably due to resolving a merge conflict the wrong way at some point. I'll put in a PR to fix.

Reply:
Sorry about messing that up, Wayne! Yes, the mistake happened probably during merging...